tests/unit/generators/schema-generator-test.php

```diff
@@ -324,6 +341,22 @@ public function test_generate_with_yoast_schema_block() {
 			],
 		];
 
+		$yoast_schema_replaced = [
+			'@id'              => 'http://example.com/#/schema/yoast-recipe',
+			'@type'            => 'Recipe',
+			'author'           => [
+				'@id' => '#author-1337',
```

Comment:
I approve

Reply:
Blatantly stole this from Andy, so he deserves all the credit 😉.